pkg/formatter/csv.go

```diff
@@ -131,12 +131,17 @@ func MakeNodesReportCSV(records []*reporting.NodeReportItem, nodeFilter string)
 		)
 		if len(cookbooksList) != 0 {
 			cookbooksString = strings.Join(cookbooksList, " ")
+			if record.GetPolicyGroup() != "no group" {
```

Comment:
I think having string comparison makes future refactors fragile. Instead, I think we should add a `HasPolicyGroup() Boolean` function

Reply:
Agreed and fixed.